shared/model/response.ts

```diff
@@ -0,0 +1,11 @@
+export default class ResponseEntity<T> {
+    success: boolean;
+    errors: Array<any>;
+    data: T;
+
+    constructor(data: T, errors: Array<any>) {
```

Comment:
Array<string>

Reply:
data?

Reply:
how will we create response without `data`?

Reply:
I think constructor is redundant. We can use it class only as type.